src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -172,7 +174,12 @@ class ChargeFeeFine extends React.Component {
     );
 
     return this.props.mutator.accounts.POST(typeAction)
-      .then(() => this.newAction({}, typeAction.id, typeAction.feeFineType, typeAction.amount, commentInfo, typeAction.remaining, 0, typeAction.feeFineOwner));
+      .then(() => this.newAction({}, typeAction.id, typeAction.feeFineType, typeAction.amount, commentInfo, typeAction.remaining, 0, typeAction.feeFineOwner))
+      .then(() => {
+        this.formSubmitting = false;
+
+        return undefined;
```

Comment:
Why do we need manually return `undefined`?

Reply:
I did it because eslint throws error - `arrow function cannot return assignment`, and also if add empty `return` - eslint error will be `no useless return`